core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1039,10 +1039,10 @@ class HttpGoogleServicesDAO(
       .build()
   }
 
-  def toProxyFromUser(rawlsUser: RawlsUser) = toProxyFromUserSubjectId(rawlsUser.userSubjectId.value)
-  def toProxyFromUser(userInfo: UserInfo) = toProxyFromUserSubjectId(userInfo.userSubjectId)
-  def toProxyFromUser(subjectId: RawlsUserSubjectId) = toProxyFromUserSubjectId(subjectId.value)
-  def toProxyFromUserSubjectId(subjectId: String) = s"PROXY_${subjectId}@${appsDomain}"
+  def toProxyFromUser(rawlsUser: RawlsUser) = toProxyFromUserSubjectIdString(rawlsUser.userSubjectId.value)
+  def toProxyFromUser(userInfo: UserInfo) = toProxyFromUserSubjectIdString(userInfo.userSubjectId.value)
+  def toProxyFromUser(subjectId: RawlsUserSubjectId) = toProxyFromUserSubjectIdString(subjectId.value)
+  def toProxyFromUserSubjectIdString(subjectId: String) = s"PROXY_${subjectId}@${appsDomain}"
```

Comment:
this seems to go slightly against the grain of being type safe- why not take in a `RawlsUserSubjectId` and extract the value inside of this function?